examples/3.audio_analysis/audio_transcription/audio_transcription.ipynb

```diff
@@ -7,11 +7,20 @@
     "# Audio transcription\n",
```

Comment:
<p>Probably we should delete this output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='47'/>